tools/region-parser/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("//tools:defaults.bzl", "ts_library")
```

Comment:
Can you add a README.md to the region-parser folder or a comment somewhere in one of these files saying how it's just copied over from the angular/angular repo?

Reply:
If we had to make any significant changes, note those too. Will be good to know about if we try to converge them at some point